ccd-definition/CaseEventToFields/CareSupervision/createCaseLink.json

```diff
@@ -0,0 +1,29 @@
+[
+  {
+    "LiveFrom": "01/01/2018",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "createCaseLink",
+    "CaseFieldID": "caseLinks",
```

Comment:
Maybe it is needed but reading 4c on the guide do we need to add this?

Reply:
Yeah, scrolling down to the example configuration it shows you need to add the CaseEvent and CaseEventToFields for both create and maintain case link events